features/admin.branding.v1/constants/preview-screen-constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const BRANDING_PREVIEW_SCREEN_ID: string = "branding.preview.screens.";
```

Comment:
Variable name suggestion;
```suggestion
// Feature keys.
export const BRANDING_PREVIEW_SCREEN_ID_PREFIX: string = "branding.preview.screens.";
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7005/commits/d71e795cadeda58c790fa54f08ada0058b336c18